website/docs/getting-started/setup/installation-guides/kubernetes/README.mdx

```diff
@@ -0,0 +1,357 @@
+| `applicationConfig.APPSMITH_CLIENT_LOG_LEVEL`            | `""`  |
+| [`applicationConfig.APPSMITH_GOOGLE_MAPS_API_KEY`](/getting-started/setup/instance-configuration/google-maps)         | `""`  |
+| [`applicationConfig.APPSMITH_MAIL_ENABLED`](/getting-started/setup/instance-configuration/email/)                | `""`  |
+| [`applicationConfig.APPSMITH_MAIL_HOST`](/getting-started/setup/instance-configuration/email/)                   | `""`  |
+| [`applicationConfig.APPSMITH_MAIL_PORT`](/getting-started/setup/instance-configuration/email/)                   | `""`  |
+| [`applicationConfig.APPSMITH_MAIL_USERNAME`](/getting-started/setup/instance-configuration/email/)               | `""`  |
+| [`applicationConfig.APPSMITH_MAIL_PASSWORD`](/getting-started/setup/instance-configuration/email/)               | `""`  |
+| [`applicationConfig.APPSMITH_MAIL_FROM`](/getting-started/setup/instance-configuration/email/)                   | `""`  |
+| [`applicationConfig.APPSMITH_REPLY_TO`](/getting-started/setup/instance-configuration/email/)                    | `""`  |
+| [`applicationConfig.APPSMITH_MAIL_SMTP_AUTH`](/getting-started/setup/instance-configuration/email/)              | `""`  |
+| `applicationConfig.APPSMITH_MAIL_SMTP_TLS_ENABLED`       | `""`  |
+| [`applicationConfig.APPSMITH_DISABLE_TELEMETRY`](https://docs.appsmith.com/product/telemetry#disable-telemetry)           | `""`  |
+| `applicationConfig.APPSMITH_RECAPTCHA_SITE_KEY`          | `""`  |
+| `applicationConfig.APPSMITH_RECAPTCHA_SECRET_KEY`        | `""`  |
+| `applicationConfig.APPSMITH_RECAPTCHA_ENABLED`           | `""`  |
+| [`applicationConfig.APPSMITH_MONGODB_URI`](/getting-started/setup/instance-configuration/custom-mongodb-redis#custom-mongodb)                | `""`  |
+| [`applicationConfig.APPSMITH_REDIS_URL`](/getting-started/setup/instance-configuration/custom-mongodb-redis#custom-redis)                   | `""`  |
+| `applicationConfig.APPSMITH_ENCRYPTION_PASSWORD`         | `""`  |
+| `applicationConfig.APPSMITH_ENCRYPTION_SALT`             | `""`  |
+| [`applicationConfig.APPSMITH_CUSTOM_DOMAIN`](/getting-started/setup/instance-configuration/custom-domain/#dockerk8s)               | `""`  |
+
+Once you have made changes to the values file, run the below command.
+
+```bash
+helm install -f values.yaml stable-appsmith/appsmith --generate-name
+```
+
+## Publish Appsmith
+
+To make your Appsmith installation accessible on the internet, you need an Ingress controller set up. For more information on Ingress controllers, see the Configure Kubernetes Cluster section. If you haven't yet installed the Helm chart, you can run the following command to install the chart, enable the ingress controller, and publish Appsmith.
+
+```bash 
+helm install appsmith/appsmith --generate-name \
+  --set ingress.enabled=true \
+  --set ingress.annotations."kubernetes\.io/ingress\.class"=nginx \
+  --set service.type=ClusterIP
+```
```

Comment:
@pratapaprasanna, can you help change this instructions to configuring Ingress in `values.yaml`, instead of advising users to do it with the `--set` arguments please?